fix: CREATE IF NOT EXISTS does not work at all (#6073)
confluentinc/ksql: docs/developer-guide/ksqldb-reference/create-stream.md

Create a new stream with the specified columns and properties.

> If the IF NOT EXISTS clause is present, the statement doesn't fail if the
> stream doesn't exist.

Reviewer: ```suggestion
If the IF NOT EXISTS clause is present, the statement won't fail if a
stream with the same name already exists.

```

Author: fixed this
